etc/templates/config/centos/5/configuration_assessment.template

```diff
@@ -0,0 +1,15 @@
+  <configuration_assessment>
```

Comment:
For every template:
- `configuration_assessment` -> `configuration-assessment`
- Remove application policies from the templates (Apache & MySQL)